torch/csrc/jit/tensorexpr/ir_printer.cpp

```diff
@@ -559,6 +559,7 @@ void print(const Expr* expr) {
   } else {
     std::cout << "(null expr)";
   }
+  std::cout << "\n";
```

Comment:
Intentional or leftover debugging?

Reply:
It's intentional actually. Otherwise, the output is not shown when used in lldb, and using in debugger was the main motivation to have this function :)